ec/ecresource/deploymentresource/schema_elasticsearch.go

```diff
@@ -167,3 +181,55 @@ func elasticsearchMonitoringSchema() *schema.Schema {
 		},
 	}
 }
+
+func elasticsearchConfig() *schema.Schema {
+	return &schema.Schema{
+		Type:             schema.TypeList,
+		Optional:         true,
+		MaxItems:         1,
+		DiffSuppressFunc: suppressMissingOptionalConfigurationBlock,
+		Description:      `Optionally define the Elasticsearch configuration options for the Elasticsearch nodes`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				// Settings
+
+				// Ignored settings for now: [ user_bundles and user_bundles ].
```

Comment:
Are these "TODOS" or are we ignoring them for another reason? Would be helpful to write that in comment for clarity.

Reply:
ah, they're informational that we're not implemented these fields yet. I'll add some more info.